src/Bundle/Maker/MakeFactory.php

```diff
@@ -268,6 +275,32 @@ private function defaultPropertiesForPersistedObject(string $class, bool $allFie
 
         $dbType = $em instanceof EntityManagerInterface ? 'ORM' : 'ODM';
 
+        // If Factory exist for related entities populate too with auto defaults
+        foreach ($metadata->associationMappings as $item) {
```

Comment:
I think I'll provide a more robust implementation in a further PR, it's time to split this clas. I think we should handle `--all-fields` here as well